plugins/org.locationtech.udig.catalog.worldimage/src/org/locationtech/udig/catalog/internal/worldimage/InMemoryCoverageLoader.java

```diff
@@ -115,36 +119,36 @@ private void updateMemoryLevel() throws IOException {
         int originalHeap = UiPlugin.getMaxHeapSize();
         if (originalHeap < 700) {
             heap = 1024;
-        } else if( originalHeap < 1500) {
+        } else if (originalHeap < 1500) {
             heap = 2048;
         } else {
             heap = originalHeap * 2;
         }
-        
+
         String os = Platform.getOS();
-        if (heap > 1024 && os == Platform.OS_WIN32) {
+        if (heap > 1024 && os.equals(Platform.OS_WIN32)) {
```

Comment:
Like it cursorPosition I'd suggest use `Platform.OS_WIN32.equals(Platform.getOS())` here